qiita_db/metadata_template/test/test_sample_template.py

```diff
@@ -114,14 +114,14 @@ def test_getitem_required(self):
         """
         self.assertEqual(self.tester['physical_specimen_location'], 'ANL')
         self.assertEqual(self.tester['collection_timestamp'],
-                         datetime(2011, 11, 11, 13, 00, 00))
+                         str(datetime(2011, 11, 11, 13, 00, 00)))
```

Comment:
Minor, but wouldn't it make more sense to just write the string, the datetime object is not needed.